input_fixtures/good_bam/input.json

```
@@ -0,0 +1,48 @@
+{
+  "node_info": {
+    "2146307a-7958-4fad-bd1f-ce2546eef5a9": {
+      "file_url": "/data_fxitures/ao8r2Vc/sample.bam",
```

```suggestion
      "file_url": "/data_fixtures/ao8r2Vc/sample.bam",
```
... but, since tests pass with the typo, it suggests that what goes here doesn't really matter. It would be better not to add 1M of binary files to the repo, unless you relly need to.

---

That's true - looking at the code closer and knowing that IGV-JS requires interaction to make a request to those files, those fixtures are never really used (hence the pass with the typo) since we only test the options.json file and not the behavior of the IGV viewer.  I'll remove the fixtures since I would side with you here.